src/Core/ConstraintValidator/TypedRegexValidator.php

```diff
@@ -87,6 +87,7 @@ private function getPattern($type)
             'message' => '/[<>{}]/i',
             'language_iso_code' => IsoCode::PATTERN,
             'language_code' => '/^[a-zA-Z]{2}(-[a-zA-Z]{2})?$/',
+            'dni_lite' => '/^[0-9A-Za-z-.]{1,16}$/U',
```

Comment:
@jolelievre No way for creating a constant ?

Reply:
As the other don't have const I preferred to do the same here (no related ValueObject to put the const) But maybe in the develop branch we can add it as I believe it's better organized

Reply:
Accepted.